packages/account/src/BiconomySmartAccount.ts

```diff
@@ -7,142 +7,167 @@ import {
   EntryPoint_v100__factory,
   SmartAccountFactory_v100,
   SmartAccountFactory_v100__factory,
-  SmartAccount_v100__factory,
+  SmartAccount_v100__factory
 } from '@biconomy/common'
 import { BiconomySmartAccountConfig } from './utils/Types'
 import { UserOperation, Transaction } from '@biconomy/core-types'
 import NodeClient from '@biconomy/node-client'
 import INodeClient from '@biconomy/node-client'
 import { IBiconomySmartAccount } from 'interfaces/IBiconomySmartAccount'
-import { SupportedChainsResponse, BalancesResponse, BalancesDto, UsdBalanceResponse, SmartAccountByOwnerDto, SmartAccountsResponse, SCWTransactionResponse } from "@biconomy/node-client"
+import { Bundler } from '@biconomy/bundler'
+import {
+  SupportedChainsResponse,
+  BalancesResponse,
+  BalancesDto,
+  UsdBalanceResponse,
+  SmartAccountByOwnerDto,
+  SmartAccountsResponse,
+  SCWTransactionResponse
+} from '@biconomy/node-client'
 
 export class BiconomySmartAccount extends SmartAccount implements IBiconomySmartAccount {
   private factory: SmartAccountFactory_v100
   private nodeClient: INodeClient
+  private accountIndex!: Number
 
   constructor(readonly biconomySmartAccountConfig: BiconomySmartAccountConfig) {
-    const { signerOrProvider, rpcUrl, epAddress, factoryAddress, bundlerUrl, paymasterUrl, nodeClientUrl } = biconomySmartAccountConfig
+    const {
+      signerOrProvider,
+      rpcUrl,
+      epAddress,
+      factoryAddress,
+      bundlerUrl,
+      paymasterUrl,
+      nodeClientUrl
+    } = biconomySmartAccountConfig
```

Comment:
Need to have discussion about these parameters. Can't resolve it in just comments.